test/unit/kubernetes-deploy/runner_test.rb

```diff
@@ -0,0 +1,43 @@
+require 'test_helper'
+
+class RunnerTest < Minitest::Test
+  def setup
+    @logger_stream = StringIO.new
+    @logger = Logger.new(@logger_stream)
+    KubernetesDeploy.logger = @logger
+  end
+
+  def teardown
+    @logger_stream.close
+  end
+
+  def test_that_it_has_a_version_number
+    refute_nil ::KubernetesDeploy::VERSION
+  end
+
+  def test_initializer
+    # TODO: Switch to --kubeconfig for kubectl shell out and pass env var as arg to Runner init
+    # Then fix this crappy env manipulation
+    original_env = ENV["KUBECONFIG"]
+    ENV["KUBECONFIG"] = "/this-really-should/not-exist"
```

Comment:
The problem now is that any of the lines below raise an error, it won't ever reach `ENV["KUBECONFIG"] = original_env`. Though I'm in favor of shipping it now as it is and following with the `--kubeconfig` PR.